src/controller/minesweeper.rb

```diff
@@ -1,6 +1,30 @@
 # frozen_string_literal: true
 
+require_relative '../model/board'
+require_relative '../utils/board_creators'
+require 'dotenv/load'
+
 # models a Controller for Minesweeper logic and interactions
 class MinesweeperController
   def initialize; end
+
+  def self.create_board(dimensions, bombs)
+    board = Board.new(dimensions, bombs)
+    if ENV['TEST']
```

Comment:
First of all, I'm sure that `ENV['TEST']` will return a string, so this will always be truthy. Also, I think that is better to manage this on the test files

Reply:
Yup, I agree with Moisés here, I think the best pattern is to have two different methods and call them accordingly from either main.rb or a test file.